plugins/modules/aci_cloud_region.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, nkatarmal-crest <nirav.katarmal@crestdatasys.com>
+# Copyright: (c) 2020, Cindy Zhao <cizhao@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_cloud_region
+short_description: Manage Cloud Providers Region (cloud:Region)
+description:
+-  Manage Cloud Providers Region on Cisco Cloud ACI.
+author:
+- Nirav (@nirav)
+- Cindy Zhao (@cizhao)
+options:
+  region:
+    description:
+    - The name of the cloud provider's region.
+    aliases: [ name ]
+    type: str
+  cloud_provider_profile_vendor:
+    description:
+    - The vendor of the controller
+    choices: [ aws ]
+    type: str
+    required: yes
+  state:
```

Comment:
do we still need 'state' parameter? I keep it just for consistency but it's never used in the code

Reply:
Yes keep it just in case.